autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/ConfigureCard.tsx

```diff
@@ -15,16 +15,29 @@ import { CourseCardArray } from '../../../types/CourseCardOptions';
 import Availability from '../../../types/Availability';
 import { formatTime } from '../../../timeUtil';
 
+// styles for faster loading spinner
+const useStyles = makeStyles({
+  fast: {
+    animationDuration: '550ms',
+  },
+});
+
```

Comment:
I was actually considering making a custom component to wrap `<CircularProgress />`. Do you think a theme override would be better?

Reply:
I think we should just make it a custom component like we've been doing to other small wrappers for material ui components rather than a theme override, seems more intuitive and allows us to start with the default `CircularProgress` if we end up wanting to use it for something else later